### PR TITLE
(docs) Create devops.md in polish

### PR DESCRIPTION
**Pull request by mathias3**
- [ x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [x ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x ] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: docs/i18n-languages/polish/devops.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+# Developer Operations at freeCodeCamp.org
+
+Dziękujemy za zainteresowanie dowiedzeniem się tym, jak robimy DevOps dla platformy na freeCodeCamp.org.
```
Comment (Qondor) [inline docs/i18n-languages/polish/devops.md]:
```suggestion
Dziękujemy za zainteresowanie się tym, jak robimy DevOps dla platformy na freeCodeCamp.org.
```

File: docs/i18n-languages/polish/devops.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+Staraliśmy się, aby język w tym przewodniku był jak najprostszy dla wszystkich. Jednak tutaj możesz znaleźć  techniczny żargon. Nie jest to wyczerpujący przewodnik po wszystkich operacjach i ma być wykorzystywany tylko jako punkt odniesienia dla zrozumienia systemów.
```
Comment (Qondor) [inline docs/i18n-languages/polish/devops.md]:
```suggestion
Staraliśmy się, aby język w tym przewodniku był jak najprostszy dla wszystkich. Znaleźć możesz tutaj jednak techniczny żargon. Nie jest to wyczerpujący przewodnik po wszystkich operacjach i ma być wykorzystywany tylko jako punkt odniesienia dla zrozumienia systemów.
```

File: docs/i18n-languages/polish/devops.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+## Jak budujemy i wdrażamy bazę danych?
+
+Nieprzerwanie budujemy i wdrażamy [`master`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master), nasz domyślny dział rozwoju na osobnym zestawie serwerów**.
+
+Zazwyczaj gałąź `master` jest łączona w gałęzią [`produkcyjny-stage`](https://github.com/freeCodeCamp/freeCodeCamp/tree/production-staging) raz dziennie i wydawana w izolowanej infrastrukturze. Nazywamy to naszą aplikacją "staging/beta".
```
Comment (Qondor) [inline docs/i18n-languages/polish/devops.md]:
```suggestion
Zazwyczaj gałąź `master` jest łączona z gałęzią [`produkcyjny-stage`](https://github.com/freeCodeCamp/freeCodeCamp/tree/production-staging) raz dziennie i wydawana w izolowanej infrastrukturze. Nazywamy to naszą aplikacją "staging/beta".
```

File: docs/i18n-languages/polish/devops.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+Jest ona identyczna z naszym środowiskiem produkcyjnym w `freeCodeCamp.org`, innym niż wykorzystującym osobny zestaw baz danych, serwerów, web-proxy, itp. Ta izolacja pozwala nam testować ciągły rozwój i funkcje w scenariuszu "production like", bez wpływu na zwykłych użytkowników platform freeCodeCamp.org.
```
Comment (Qondor) [inline docs/i18n-languages/polish/devops.md]:
```suggestion
Jest ona identyczna z naszym środowiskiem produkcyjnym we `freeCodeCamp.org`, innym niż wykorzystującym osobny zestaw baz danych, serwerów, web-proxy, itp. Ta izolacja pozwala nam testować ciągły rozwój i funkcje w scenariuszu "production like", bez wpływu na zwykłych użytkowników platform freeCodeCamp.org.
```

File: docs/i18n-languages/polish/devops.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+Stosujemy różne poziomy testów integracyjnych i akceptacyjnych, aby sprawdzić jakość kodu. Wszystkie nasze testy są wykonywane za pomocą oprogramowania, takiego jak Travis i Azure Pipelines. Niektóre z tych zautomatyzowanych, czyli po przeniesieniu zmian do odpowiedniego oddziału, są budowane i wdrażane na platformach.
+
+Zapraszamy do przetestowania tych wydań w trybie **"public beta testing "** i uzyskania wczesnego dostępu do nadchodzących funkcji na platformach. Czasami te funkcje/zmiany są nazywane **następnymi, beta, etapami, ** itd. zamiennie.
+
+Twój wkład poprzez opinie i raporty o wydaniach pomoże nam uczynić platformy produkcyjne na `freeCodeCamp.org` bardziej **odpornymi**, **zgodnymi** i **stabilnymi** dla każdego.
+
+Dziękujemy Ci za zgłaszanie błędów, które napotykasz i pomagamy w ulepszaniu freeCodeCamp.org. Rządzisz!
```
Comment (Qondor) [inline docs/i18n-languages/polish/devops.md]:
```suggestion
Dziękujemy Ci za zgłaszanie błędów, które napotykasz, pomagasz w ten sposób w ulepszaniu freeCodeCamp.org. Rządzisz!
```

File: docs/i18n-languages/polish/devops.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+
+## Identyfikacja nadchodzącej wersji platformy
+
+Nazwa domeny będzie inna niż **`freeCodeCamp.org`**. Obecnie ta publiczna wersja beta testowa jest dostępna na stronie internetowej:
+
+<h3 align="center"><a href='https://www.freecodecamp.dev' _target='blank'><code>www.freecodecamp.dev</code></a></h4>>
+
+Zespół dev-team łączy zmiany z gałęzi `master` do `production-stage`, gdy tylko je wypuszczą. Zazwyczaj górny commit powinien być tym, co widzisz na żywo na stronie. Możesz określić dokładną wersję wdrożonej wersji, odwiedzając dzienniki budowy i wdrażania dostępne poniżej w sekcji status.
```
Comment (Qondor) [inline docs/i18n-languages/polish/devops.md]:
```suggestion
Zespół dev-team łączy zmiany z gałęzi `master` do `production-stage` gdy tylko je wypuszczą. Zazwyczaj górny commit powinien być tym, co widzisz na żywo na stronie. Możesz określić dokładną wersję wdrożonej wersji odwiedzając dzienniki budowy i wdrażania dostępne poniżej w sekcji status.
```
"górny"? Was it "top commit" in the original version? Then it should be something like "ostatni" maybe.

File: docs/i18n-languages/polish/devops.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+
+## Identyfikacja aktualnej wersji platformy
+
+**Aktualna wersja platformy jest zawsze dostępna na stronie [`freeCodeCamp.org`](https://www.freecodecamp.org).**
+
+Zespół dev-team łączy zmiany z gałęzi `production-stage` na `production-current`, gdy tylko je wypuszczą. Górne commit powinno być to, co widzisz na żywo na stronie. Możesz określić dokładną wersję wdrożonej wersji, odwiedzając dzienniki budowy i wdrażania dostępne poniżej w sekcji statusu.
```
Comment (Qondor) [inline docs/i18n-languages/polish/devops.md]:
```suggestion
Zespół dev-team łączy zmiany z gałęzi `production-stage` na `production-current`, gdy tylko je wypuszczą. Górne commit powinno być to, co widzisz na żywo na stronie. Możesz określić dokładną wersję wdrożonej wersji, odwiedzając dzienniki budowy i wdrażania dostępne poniżej w sekcji statusu.
```
Again, "top commit" =/= "górny commit".

File: docs/i18n-languages/polish/devops.md [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+
+## Status budowania i wdrażania
+
+Używamy Azure Pipelines i innego oprogramowania CI (Travis, GitHub Actions) do ciągłego testowania i wdrażania naszych aplikacji.
+
+#### Wyzwalanie budowy
+
+Obecnie tylko zespół programistów może przesuwać się do oddziałów produkcyjnych z powodu zautomatyzowanego wdrażania na żywo. Zmiany w oddziałach `production-*` mogą wylądować tylko poprzez szybkie łączenie się z [`upstream`](https://github.com/freeCodeCamp/freeCodeCamp).
+
+> Uwaga: W nadchodzących dniach usprawnilibyśmy ten przepływ, który można osiągnąć dzięki prośbom o wyciągnięcie, w celu lepszego zarządzania dostępem i przejrzystości.
```
Comment (Qondor) [inline docs/i18n-languages/polish/devops.md]:
```suggestion
> Uwaga: W nadchodzących dniach usprawnimy ten przepływ, który można osiągnąć dzięki prośbom o wyciągnięcie, w celu lepszego zarządzania dostępem i przejrzystości.
```

File: docs/i18n-languages/polish/devops.md [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+
+2. Upewnij się, że twój mistrz jest nieskazitelnie czysty i zsynchronizowany z prądem.
+
+   ````sh
+   git checkout master
+   git fetch --all --prune -all --prune
+   git reset --hard upstream/master
+   ```
+
+3. Sprawdzić, czy urządzenie Travis CI przechodzi przez odgałęzienie główne w górę rzeki.
```
Comment (Qondor) [inline docs/i18n-languages/polish/devops.md]:
```suggestion
3. Sprawdź, czy urządzenie Travis CI przechodzi przez odgałęzienie główne w górę rzeki.
```

File: docs/i18n-languages/polish/devops.md [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+   - [ ] https://travis-ci.org/freeCodeCamp/freeCodeCamp/branches should be Green.
+
+   ![Image - Check build status on Travis CI](/docs/images/devops/travis-build.png)
+
+4. Potwierdź, że jesteś w stanie zbudować repozytorium lokalnie.
+
+   ```
+   npm run clean-and-develop (Czyste i rozwojowe)
+   ```
+5. Przesunięcie zmian z fazy master na fazę produkcji
```
Comment (Qondor) [inline docs/i18n-languages/polish/devops.md]:
```suggestion
5. Przesuń zmianę z fazy master na fazę produkcji
```

**Review by Qondor — CHANGES_REQUESTED** [review]

**Comment by mathias3** [comment]
changec included in make Contributing.md in polish #37350

**Pull request closed without merging**